test/e2e/config/functional.conf.js

```diff
@@ -1,33 +1,71 @@
-const common = require('./support/common.conf');
-const localConfig = require('./support/local.conf')
-const jenkinsConfig = require('./support/jenkins.conf')
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const minimist = require('minimist');
 
 var screenShotUtils = require("protractor-screenshot-utils").ProtractorScreenShotUtils;
 
 
-const cap = (common.argv.local) ? localConfig : jenkinsConfig;
+chai.use(chaiAsPromised);
+
+const argv = minimist(process.argv.slice(2));
+
+const jenkinsConfig = [
+
+    {
+        browserName: 'chrome',
+        acceptInsecureCerts: true,
+        nogui: true,
+        chromeOptions: { args: ['--headless', '--no-sandbox', '--disable-dev-shm-usage', '--disable-setuid-sandbox', '--no-zygote ', '--disableChecks'] }
+    }
+];
+
+const localConfig = [
+    {
+
+        browserName: 'chrome',
+        acceptInsecureCerts: true,
+        chromeOptions: { args: ['--headless', '--no-sandbox', '--disable-dev-shm-usage', '--disable-setuid-sandbox', '--no-zygote '] },
+        proxy: {
+            proxyType: 'manual',
+            httpProxy: 'proxyout.reform.hmcts.net:8080',
+            sslProxy: 'proxyout.reform.hmcts.net:8080',
+            noProxy: 'localhost:3000'
+        }
+    }
+];
+
+const cap = (argv.local) ? localConfig : jenkinsConfig;
 
 const config = {
+    SELENIUM_PROMISE_MANAGER: false,
     framework: 'custom',
     frameworkPath: require.resolve('protractor-cucumber-framework'),
-    specs: common.specFilesFilter,
+    specs: ['../features/**/*.feature'],
     baseUrl: process.env.TEST_URL || 'http://localhost:3000/',
-
     params: {
         serverUrls: process.env.TEST_URL || 'http://localhost:3000/',
-        targetEnv: common.argv.env || 'local'
-    },
+        targetEnv: argv.env || 'local',
+        username: 'lukesuperuserxui@mailnesia.com',
```

Comment:
we have usernames and passwords that are being committed?

Reply:
@sreekanthpuligadda  can you look at this

Reply:
@paddybasi @udaydenduluri33  these test accounts are in protractor config files from beginning. We can move them out of code base, however this would require some planning and refactoring effort to cleanup. Can this be done as a separate story outside this sync activity?

Reply:
Ok